README.md

```diff
@@ -1 +1,99 @@
-# lookout-sdk
\ No newline at end of file
+lookout SDK [![GoDoc](https://godoc.org/gopkg.in/src-d/lookout-sdk.v0?status.svg)](https://godoc.org/github.com/src-d/lookout-sdk) [![PyPI version](https://badge.fury.io/py/lookout-sdk.svg)](https://pypi.org/project/lookout-sdk/) [![Build Status](https://travis-ci.org/src-d/src-d/lookout-sdk.svg)](https://travis-ci.org/src-d/src-d/lookout-sdk)
+-----------
+
+What is lookout SDK?
+===================
+Lookout SDK is toolkit for writing new analyzers for the [lookout](https://github.com/src-d/lookout/).
+
+
+What does it provide
+====================
+It provides to analyzer an easy access to the DataService API though gRPC service.
+
+DataService abstracts all data access and details of dealing witn actual Git repositories, UAST extraction, programming language detection, etc. This way analyzer can only focus on source code analysis logic.
+
+Architecture of lookout and it's componetes are described in [src-d/lookout/docs](https://github.com/src-d/lookout/tree/master/docs#lookout)
+
+SDK includes:
+ - proto definitinos
+ - pre-generate libraries code for Golang and Python
+ - quickstart documentation on how to write an anlyzer
+
+
+How to create a new Analyzer
+============================
+
+Essentially, every analyzer is a [gRCP server](https://grpc.io/docs/guides/#overview) that implements [Analyzer service](./proto/service_analyzer.proto#L30). Lookout itself act as a gRCP client and will push your analyzer whenever a new PR is ready for anlysis.
```

Comment:
Lookout itself **acts**
whenever a new PR -> I'd use Pull Request
ready for **analysis**